src/commands/configure/node.rb

```diff
@@ -18,10 +18,21 @@ def answers_file
           config.node_answers_file(node_name)
         end
 
+        def higher_level_answer_files
+          [
+            config.domain_answers_file,
+            config.group_answers_file(node.primary_group)
+          ]
+        end
```

Comment:
These need to be included as an optional dependency. It can probably be placed in `ConfigureCommand` so the inherited commands do not need to worry about it